packages/discovery/src/peer-exchange/waku_peer_exchange_discovery.ts

```diff
@@ -102,6 +106,8 @@ export class PeerExchangeDiscovery
       return;
     }
 
+    this.dispatchEvent(new CustomEvent("status", { detail: true }));
```

Comment:
let's figure out if we really need to introduce this event (I am hoping to reuse `peer` event)

but in case we end up I'd rename it to something more concrete as it is not clear to me what does `status` mean and what `true`/`false` is about tbh

Reply:
in that case it seems that we can use `isStarted` property that is already present (only need to make it public):
https://github.com/waku-org/js-waku/blob/9b0f1e855aa3a1f7b9aec3a4c726568d37595c28/packages/discovery/src/peer-exchange/waku_peer_exchange_discovery.ts#L64  

also we need to actually set it to true as it is never set in [start](https://github.com/waku-org/js-waku/blob/9b0f1e855aa3a1f7b9aec3a4c726568d37595c28/packages/discovery/src/peer-exchange/waku_peer_exchange_discovery.ts#L100)

would it work in that case, @danisharora099 ?